insights/tests/client/connection/test_test_connection.py

```diff
@@ -0,0 +1,76 @@
+from logging import ERROR
+from unittest.mock import patch
```

Comment:
This only works in Python3 , you need to
```py
try:
    from unittest.mock import patch
except:
    from mock import patch
```

Reply:
```python
from mock.mock import patch
```

should work both for Python 2 and 3. It’s already used in other tests, e.g. [test_checkin.py](https://github.com/RedHatInsights/insights-core/blob/a27d1f52a146d04b78047455c9741d487ae4dbf9/insights/tests/client/connection/test_checkin.py#L4).